src/gobapi/graphql_streaming/response_custom.py

```diff
@@ -38,8 +39,8 @@ def parse_list(l):
             items = [s.strip() for s in items]
             return [s for s in items if s]
 
-        self.flatten = request_args.get('flatten') or False
-        self.lowercase = request_args.get('lowercase') or False
+        self.flatten = strtobool(request_args.get('flatten') or 'false')
+        self.lowercase = strtobool(request_args.get('lowercase') or 'false')
```

Comment:
strtobool doesn't really make a bool, but rather an integer 1 or 0, better do bool(strtobool(....)). Would also prefer to write request_args.get('flatten', 'false').

Reply:
Totally agree. I kept the 'or' intact because it was in the original approach. And, indeed, better be explicit about the boolean conversion.